src/ceph-detect-init/integration/test_main.py

```diff
@@ -54,10 +54,10 @@ def run(os):
 class TestCephDetectInit(testtools.TestCase):
 
     def test_centos_6(self):
-        self.assertEqual('sysvinit', run('centos-6'))
+        self.assertEqual('upstart', run('centos-6'))
```

Comment:
Aye. CentOS 6 shipped Upstart, since it was the default init system in RHEL 6 release. I also tested this on a Docker container.
```sh
$ init --version
init (upstart 0.6.5)
```